pkg/cmd/pulumi/deployment_settings_config.go

```diff
@@ -612,23 +646,31 @@ func configureGitPassword(ctx context.Context, be backend.Backend,
 	return nil
 }
 
-func configureGitSSH(ctx context.Context, be backend.Backend, s backend.Stack,
-	sd *workspace.ProjectStackDeployment, gitSSHPrivateKeyPath string,
-) error {
+func configureGitSSH(d *deploymentSettingsCommandDependencies, gitSSHPrivateKeyPath string) error {
 	if gitSSHPrivateKeyPath == "" {
-		return errors.New("No SSH private key was provided")
+		configureMsg := "No SSH private key was provided, run `pulumi deployment settings " +
+			"configure` with the `--git-ssh-private-key` flag set"
+		configureMsg = colors.Highlight(configureMsg, "No SSH private key was provided", colors.Red+colors.Bold)
+		configureMsg = colors.Highlight(configureMsg, "pulumi deployment settings configure", colors.BrightBlue+colors.Bold)
+		configureMsg = colors.Highlight(configureMsg, "git-ssh-private-key", colors.BrightBlue+colors.Bold)
```

Comment:
Does this need newlines, or some explanation of what the commands are doing? It's not super clear to me what's going on here.

Reply:
Ignore me -- I've seen what `Highlight` does. Not sure I'm a fan of this pattern, but hey ho. On my list to kill.

Reply:
yes it is sort of odd, but I am sticking to what is already being used in the cli